Monika After Story/game/script-topics.rpy

```diff
@@ -15990,3 +15991,118 @@ label monika_stargazing:
+    return
+
+# Cosmetic Glasses
+label monika_player_eyesight_cg:
+    $ persistent._mas_pm_wears_glasses = False
+    $ persistent._mas_pm_wears_lenses = False
+    $ persistent._mas_pm_poor_eyesight = False
+    m 5ekbsa "Aw, [player], I wish I really could see your face with them on."
+    m 5hubsa "I bet you look so smart, intelligent and confident~"
+    m 1sua "Maybe I could even get myself a pair and try them out, too, ehehe."
+    m 1tuu "What do you think, [mas_get_player_nickname()]? Would they suit me well, hmm?"
+    return
+
+# Eyesight Aids When Needed
+label monika_player_eyesight_eawn:
+    m 3rka "I see... But [player], is it, by any chance, because of your eyesight?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "I see... But [player], is it, by any chance, because of your eyesight?{fast}"
+
+        "Yes...":
+            $ persistent._mas_pm_poor_eyesight = True
+            m 2ekc "Oh, it's sad to hear that you're having problem with your eyes."
+            m 2esa "I really hope wearing glasses helps you improve it."
+            m 2hua "Thank you for caring about yourself, [player]."
+            m 3lsbsb "This world has so much to see, and I'd like to see everything together with you~"
+
+        "No.":
+            $ persistent._mas_pm_poor_eyesight = False
+            m 1rusdlb "Ahaha, sorry for assuming then."
+            m 1rsbssdlb "I was just a bit worried since you spend a lot of time here with me, on your computer..."
+            m 1ekc "And from what I heard it can really hurt your eyes if you sit in front of it for too long."
+            m 3eka "So, [player], even if you're good... Don't neglect your health, okay?"
+            if mas_seenLabels(["monika_using_pcs_healthily"]):
+                m 1eub "Remember me talking about doing frequent breaks if you're using your computer for a while?"
+                m 1esa "You could take a little break every 15-20 minutes and let your eyes rest."
+                if mas_isMoniLove(higher=True):
+                    m 1rsa "Say, you could close them, relax and think about something for a few minutes... "
+                    m 1dsa "Maybe you could even dream... "
+                    extend 1dsbsa  "you know... "
+                    extend 1tubsu "of me, ehehe."
+                    m 3eubsb "Just don't get too distracted, alright?~"
+                else:
+                    m 1rsa "Say, you could close them and dream of something for a few minutes."
+                    m 3eua "But if you can't, you could just get distracted a bit and take a look around or glance at the window."
+                    m 1eub "That would really help, too!"
+            else:
+                $ pronoun = {"M": "his", "F": "her", "X": "their"}.get(persistent.gender, "his")
+                m 1hubsa "I just want my [mas_get_player_nickname(exclude_names=['my love'])] to have no problem with [pronoun] eyes~"
+                $ pushEvent("monika_using_pcs_healthily")
```

Comment:
I don't think we need push this.

Reply:
How about we queue it instead? My idea was to let Monika talk about player using their computer with less health impact if she didn't yet.

Reply:
This will need some kind of leadin for that topic. Also people already complain about running out of topics quickly, I don't want them to consume 2 topics at once, especially when there's no real need in it.